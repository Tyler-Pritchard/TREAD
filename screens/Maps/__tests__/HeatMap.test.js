import React from 'react';
import {shallow} from 'enzyme';
import Heatmap from '../HeatMapView';


describe('Heatmap', () => {
    describe('Rendering', () => {
        it('should match to snapshot', () => {
            const component = shallow(<Heatmap label="test label"/>)
            expect(component).toMatchSnapshot()
        });
    });
});


// test.only('FIRST TEST THAT RUNS', () => {
//     expect(true).toBe(true);
//   });
  
//   test('LAST TEST WILL NOT RUN', () => {
//     expect('A').toBe('A');
//   });



//it renders default elements
//it renders marker1
//it handles valid input submission
//it renders marker2
//it renders polyline