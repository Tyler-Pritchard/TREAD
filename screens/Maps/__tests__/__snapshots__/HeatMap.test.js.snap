// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heatmap Rendering should match to snapshot 1`] = `ShallowWrapper {}`;
